apps/bond/components/BondingProducts/Bonding/BondingList/BondingList.jsx

```diff
@@ -86,11 +90,17 @@ const getColumns = (onClick, isActive, acc, depositoryAddress, hideEmptyProducts
       dataIndex: 'fullCurrentPriceLp',
       key: 'fullCurrentPriceLp',
       width: 140,
-      render: (x, details) => (
-        <a href={details.currentPriceLpLink} rel="noopener noreferrer" target="_blank">
-          {x}
-        </a>
-      ),
+      render: (x, details) => {
+        if (isCurrentPriceLpZero(x)) {
+          return <Loader />;
+        }
+
+        return (
+          <a href={details.currentPriceLpLink} rel="noopener noreferrer" target="_blank">
+            {x}
```

Comment:
hard to understand what `x` is, considering that we didn't move to typescript. do you think it's possible to give proper names?

Reply:
ahh sorry forgot :P